packages/components/src/range-control/index.tsx

```diff
@@ -35,67 +35,79 @@ import {
 	Wrapper,
 } from './styles/range-control-styles';
 
+import type { RangeControlProps } from './types';
+import type { WordPressComponentProps } from '../ui/context';
+
 const noop = () => {};
 
-function RangeControl(
-	{
+function UnforwardedRangeControl< IconProps = unknown >(
+	props: WordPressComponentProps<
+		RangeControlProps< IconProps >,
+		'div',
```

Comment:
This is an interesting consequence of how we type and forward refs in this component — normally, I'd expect this component to inherit props from the HTML `input` element (like, for example, [`UnitControl` does](https://github.com/WordPress/gutenberg/blob/33d84b036592a5bf31af05b7710f3b2b14163dc4/packages/components/src/unit-control/index.tsx#L40-L43))

@mirka , what you do think?

Reply:
If I'm following the code correctly, it _does_ look like it's supposed to inherit from `input`. That's where the `otherProps` ultimately go.

```suggestion
		'input',
```

(It's funny because `div` and `span` are perhaps the only two mistakes you can make in this context without TS throwing an error 😂)